### ISSUE TITLE
Mark fulfilled needs

### ISSUE THREAD

**Issue by IamIsPra** [reliefsupports/reliefsupports.org-v1#12]

**Comment by THPubs** [comment]
Yes, this is a critical option. With that, we can put the fulfilled needs to the bottom of the list and put more focus on the others. Also, I suggest we add a reminder to the user to mark it as fulfilled if possible.

**Comment by arunoda** [comment]
May be it's pretty hard to fulfill some of the needs at once. So may be we can show how much of donations/stuff went to that.
Also we may need a set of admins to audit these posts and see whether they fulfilled.

Just like needs, we need the same for donations.

**Comment by THPubs** [comment]
Agreed. I can help with the audit work. Won't be able to do much with the
app because I'm not familiar with Laravel.

Anyway I think we better have a quick discussion on what to do. Rain might
start again so we have to do something fast.

We can also use this site to warn people of eminent floods. There's much
more we can do...

**Comment by danishka** [comment]
One of my idea was get volunteers to call for critical requirements.
Most of social media posts keep spreading days and days but there is no follow up.
Its hard but we can give a try if people won't return to us and inform it has been completed.

**Comment by niroshubey966** [comment]
Yeah. I think We can make a volunteers's network for critical requirements. By using that we can support to solve these problems.

**Comment by danishka** [comment]
Today one of my friend call one of the critical request. It was about a boat and they have thanked for calling and it was already sorted out. Request 43.

**Comment by danishka** [comment]
have a look on this as well https://github.com/reliefsupports/reliefsupports.org/issues/75

**Comment by danishka** [comment]
We need to add
Who call back (username)?
Time and Date?
Request has been fulfilled or not.
Comment

**Comment by isuru4lk** [comment]
Since we don't have any authentication yet either we've to implement authentication first or we've to build a small admin panel that we can mark the completed items.

If we're going to implement this after authentication of course we've to give the access to the user who has added the need plus the admin/moderator.

**Comment by danishka** [comment]
We started working on authentication under release 0.4

**Comment by danishka** [comment]
@THPubs spoke to @isuru4lk and he will take over this.
But you can add ideas.

Please note that requests for lunch packets must be alerted when it getting delay to attend and it should not display again but can be show as missed deliveries on yesterday. 
Then someone should make a call and confirm those people really starving or someone already delivered food.

**Comment by danishka** [comment]
may be we can send them to archived_requests table and still available to access in a different page/tab.

**Comment by danishka** [comment]
More details on #155
Have a look
